ui/src/app/settings/containers/Users/Users.js

```diff
@@ -14,54 +15,85 @@ import Pagination from "app/base/components/Pagination";
 import MainTable from "app/base/components/MainTable";
 import Row from "app/base/components/Row";
 
-const generateUserRows = (users, authUser) =>
-  users.map(user => ({
-    columns: [
-      {
-        content: <Link to={`/users/${user.id}`}>{user.username}</Link>,
-        role: "rowheader"
-      },
-      { content: user.email },
-      { content: 37, className: "u-align--right" },
-      { content: "Local" },
-      { content: "12 mins ago" },
-      { content: user.is_superuser ? "Admin" : null },
-      { content: user.sshkeys_count, className: "u-align--right" },
-      {
-        content: (
-          <>
-            <Link
-              to={`/users/${user.id}/edit`}
-              className="p-button--base is-small"
-            >
-              <i className="p-icon--edit">Edit</i>
-            </Link>
-            {user.id !== authUser.id && (
-              <Link
-                to={`/users/${user.id}/delete`}
-                className="p-button--base is-small"
+const generateUserRows = (users, authUser, expandedId, setExpandedId) =>
+  users.map(user => {
+    const expanded = expandedId === user.id;
+    return {
+      className: expanded ? "user-list__expanded-row" : null,
+      columns: [
+        {
+          content: <Link to={`/users/${user.id}`}>{user.username}</Link>,
+          role: "rowheader"
+        },
+        { content: user.email },
+        { content: 37, className: "u-align--right user-list__cell--small" },
+        { content: "Local", className: "user-list__cell--small" },
+        { content: "12 mins ago" },
+        {
+          content: user.is_superuser ? "Admin" : null,
+          className: "user-list__cell--small"
+        },
+        {
+          content: user.sshkeys_count,
+          className: "u-align--right user-list__cell--small"
+        },
+        {
+          content: (
+            <>
```

Comment:
Is this meant to have an empty element?

Reply:
This is shorthand for `React.Fragment` which can be used when you need to wrap multiple elements but don't need anything to appear in the DOM.